tests/failpoints/cases/test_disk_full.rs

```diff
@@ -0,0 +1,209 @@
+// Copyright 2021 TiKV Project Authors. Licensed under Apache-2.0.
+
+use raftstore::store::msg::*;
+use std::time::Duration;
+use std::time::*;
+use test_raftstore::*;
+use tikv_util::config::*;
+/*
+Disk full test will have 3 scenarios.
+[leader full]
+1. not allowed: business wirte, region merge/split,etc.
+2. allowed: read, config change(add/remove peer), transfer leader etc.
+
+[minority full]
+1. not allowed: log entry append
+2. allowed: config change,transfer leader,etc.
+
+[majority full]
+1. equal to minority full, no additional design.
+*/
+const DISK_FULL_LEADER: &str = "disk_full_leader";
+const DISK_FULL_FOLLOWER_1: &str = "disk_full_follower_1";
+
+fn prepare_test_data(cluster: &mut Cluster<ServerCluster>) {
```

Comment:
It's unnecessary and can be removed.

Reply:
should kept, because the merge region case need.